src/components/StatsCard/StatsCard.test.tsx

```diff
@@ -1,45 +1,41 @@
-import { Edit } from '@mui/icons-material';
 import { ThemeProvider } from '@mui/material/styles';
-import { render, screen } from '@testing-library/react';
+import { render, screen, cleanup } from '@testing-library/react';
 import React from 'react';
-import { describe, it, expect } from 'vitest';
+import { describe, it, expect, afterEach } from 'vitest';
```

Comment:
You added import of cleanup from @testing-library/react. Recent versions of @testing-library/react configure automatic cleanup between tests; manually calling cleanup is often unnecessary and can be redundant.

**Sugerencia:**
```typescript
If your test environment doesn't auto-cleanup, keep this. Otherwise remove the explicit cleanup import and rely on the testing-library automatic cleanup. Also ensure consistency with other tests in the project.
```